installers/rpm-code/codes/raydata.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+raydata_main() {
+    codes_dependencies common
+    install_pip_install \
+        ModestImage \
+        area_detector_handlers \
+        databroker-pack \
+        git+https://github.com/NSLS-II/eiger-io.git \
+        hdf5plugin \
+        pyOlog \
+        pychx \
+        xray-vision
+
+    raydata_patch
+}
+
+raydata_patch() {
+    local p=$(python -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')
```

Comment:
This is available via `codes_python_lib_dir`.

Reply:
94010b4